src/main.rs

```diff
@@ -8,4 +8,6 @@ fn foo(iter: impl Iterator<Item = u32>) {
     scope(move || for _i in iter {});
 }
 
-fn main() {}
+fn main() {
+    println!("hello world!");
```

Comment:
i made this change because i wanted to say hello to the world

Reply:
makes sense!